<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications, software development and scientific communication</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 54px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 59px;
  margin-top: -59px;
}

.section h2 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h3 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h4 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h5 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h6 {
  padding-top: 59px;
  margin-top: -59px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Magali Richard</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="about.html">
    <span class="fa fa-user"></span>
     
    About
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-code-fork"></span>
     
    Research
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-line-chart"></span>
     
    Teaching
  </a>
</li>
<li>
  <a href="publication.html">
    <span class="fa fa-institution"></span>
     
    Publication
  </a>
</li>
<li>
  <a href="cv.html">
    <span class="fa fa-file-pdf-o"></span>
     
    CV
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:magali.richard@univ-grenoble-alpes.fr">
    <span class="fa fa-envelope-o"></span>
     
    Contact me
  </a>
</li>
<li>
  <a href="https://twitter.com/magg_richard">
    <span class="fa fa-twitter"></span>
     
    Twitter
  </a>
</li>
<li>
  <a href="https://github.com/magrichard">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications, software development and scientific communication</h1>

</div>


<p><a href="https://orcid.org/0000-0003-3165-3218">ORCID</a></p>
<div id="research-articles" class="section level2">
<h2>Research articles</h2>
<p>[6] Salignon #, J., <strong>Richard #, M.</strong>, Fulcrand, E., Duplus-Bottin, H. &amp; Yvert, G. (2018) Unexpected selection in fluctuating environments. In press at Molecular Systems Biology. <a href="https://doi.org/10.1101/129163">Click</a> or <a href="129163.full.pdf">PDF</a></p>
<p>[5] <strong>Richard, M.</strong>, Chuffart, F, Duplus-Bottin, H., Pouyet, F., Spichty, M., Fulcrand, E., Entrevan, M., Barthelaix, A., Springer, M., Jost, D., and G. Yvert , G. (2018). Assigning function to natural allelic variation via dynamic modeling of gene network induction. Molecular Systems Biology, 14, e7803. <a href="https://doi.org/10.15252/msb.20177803">Click</a> or <a href="msbgal3.full.pdf">PDF</a></p>
<p>[4] Chuffart, F., <strong>Richard, M.</strong>, Jost, D., Burny, C., Duplus-Bottin, H., Ohya, Y., &amp; Yvert, G. (2016). Exploiting Single-Cell Quantitative Data to Map Genetic Variants Having Probabilistic Effects. PLoS Genetics, 12(8), e1006213. <a href="http://doi.org/10.1371/journal.pgen.1006213">Click</a> or <a href="PLoS%20Genetics%202016%20Chuffart.pdf">PDF</a></p>
<p>[3] <strong>Richard, M.</strong> &amp; Yvert, G. L. How does evolution tune biological noise? Front. Genet. 5, 1011 (2014). <a href="http://doi.org/10.3389/fgene.2014.00374">Click</a> or <a href="frontiers.pdf">PDF</a></p>
<p>[2] <strong>Richard, M.</strong>, Boulin, T., Robert, V. J. P., Richmond, J. E., &amp; Bessereau, J.-L. (2013). Biosynthesis of ionotropic acetylcholine receptors requires the evolutionarily conserved ER membrane complex. Proceedings of the National Academy of Sciences of the United States of America, 110(11), E1055–63. <a href="http://doi.org/10.1073/pnas.1216154110">Click</a> or <a href="pnas.pdf">PDF</a></p>
<p>[1] Legent, K., Steinhauer, J., <strong>Richard, M.</strong>, &amp; Treisman, J. E. (2012). A screen for X-linked mutations affecting Drosophila photoreceptor differentiation identifies Casein kinase 1α as an essential negative regulator of wingless signaling. Genetics, 190(2), 601–616. <a href="http://doi.org/10.1534/genetics.111.133827">Click</a> or <a href="genetics.pdf">PDF</a></p>
</div>
<div id="r-packages" class="section level2">
<h2>R packages</h2>
<p>4/ <a href="https://github.com/bcm-uga/dmprocr">dmprocr</a> (2017) P. Terzian, F. Chuffart, D. Jost &amp; <strong>M. Richard</strong>. R package to perform differential methylation profiles clustering.</p>
<p>3/ <a href="https://github.com/bcm-uga/penda">penda</a> (2017) D. Jost &amp; <strong>M. Richard</strong>. R package to calculate personalized differential analysis of omics data. En développement.</p>
<p>2/ <a href="https://github.com/magrichard/facsticor">facsticor</a> (2015) D. Jost &amp; <strong>M. Richard</strong>. R package to calculate personalized differential analysis of omics data.</p>
<p>1/ <a href="https://github.com/magrichard/h4decoder5">h4decoder5</a> (2013) <strong>M. Richard</strong>. R package to generate error-correcting DNA barcodes.</p>
</div>
<div id="conferences" class="section level2">
<h2>Conferences</h2>
<div id="section" class="section level4">
<h4>2017</h4>
<p>Epigenetics and Cancer, Poster, Heidelberg, Allemagne.<em>Identifying epigenetic hotspots in lung cancer</em></p>
<p>Advanced lecture course on computational systems biology, Talk, Aussois, France. <em>Functional Genetic Diversity of the Yeast Galactose Network</em></p>
</div>
<div id="section-1" class="section level4">
<h4>2016</h4>
<p>JOBIM, Poster, Lyon, France. <em>Functional Genetic Diversity of the Yeast Galactose Network</em></p>
</div>
<div id="section-2" class="section level4">
<h4>2015</h4>
<p>Design optimization and control in system and synthetic biology, Poster, Paris, France. <em>Functional Genetic Diversity of the Yeast Galactose Network</em></p>
<p>BCM Seminar, Invited speaker, Grenoble, France. <em>Functional Genetic Diversity of the Yeast Galactose Network</em></p>
<p>JOBIM, Poster, Clermont-Ferrand. <em>Identification of fitness advantage in dynamic environment by modelisation of BAR-seq data</em></p>
<p>Quatrièmes Rencontres R, Talk, Grenoble, France. <em>A general method to map single-cell probabilstic trait loci of the genome</em></p>
</div>
<div id="section-3" class="section level4">
<h4>2014</h4>
<p>Experimental Approaches to Evolution and Ecology using Yeast and other Model Systems, Poster, Heidelberg, Germany. <em>Genetics of fitness in dynamic environments</em></p>
<p>Single cells genomics, Poster, Stockholm, Sweden. <em>Competition between cells in dynamic environments</em></p>
</div>
<div id="section-4" class="section level4">
<h4>2012</h4>
<p>Young Researcher in Life Sciences, Talk, 3rd prize, Paris, France. <em>Control of acetylcholine receptor maturation and ER homeostasis by a new transmembrane complex</em></p>
<p>LBMC Seminar, Invited speaker, Lyon, France. <em>Control of acetylcholine receptor maturation and ER homeostasis by a new transmembrane complex</em></p>
<p>Ver midi, Talk, Paris, France. <em>Control of acetylcholine receptor maturation and ER homeostasis by a new transmembrane complex</em></p>
</div>
<div id="section-5" class="section level4">
<h4>2011</h4>
<p>18th International Worm Meeting, Poster, Los Angeles, USA. <em>Control of acetylcholine receptor maturation and ER homeostasis by a new transmembrane complex</em></p>
</div>
<div id="section-6" class="section level4">
<h4>2010</h4>
<p>Neuronal development, synaptic function and behaviour in C. elegans, Poster, Ma- dison, USA. <em>TMEM-93 is a Conserved Transmembrane Protein Required for Assembly or Stability of Acetylcholine Receptors</em></p>
</div>
<div id="section-7" class="section level4">
<h4>2009</h4>
<p>17th International Worm Meeting, Poster, Los Angeles, USA. <em>TMEM-93 is a Conserved Transmembrane Protein Required for Assembly or Stability of Acetylcholine Receptors</em></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
