<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Research projects</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<link rel="stylesheet"
      href="site_libs/highlight/textmate.css"
      type="text/css" />
<script src="site_libs/highlight/highlight.js"></script>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 54px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 59px;
  margin-top: -59px;
}

.section h2 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h3 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h4 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h5 {
  padding-top: 59px;
  margin-top: -59px;
}
.section h6 {
  padding-top: 59px;
  margin-top: -59px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Magali Richard</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="about.html">
    <span class="fa fa-user"></span>
     
    About
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-code-fork"></span>
     
    Research
  </a>
</li>
<li>
  <a href="teaching.html">
    <span class="fa fa-birthday-cake"></span>
     
    Teaching
  </a>
</li>
<li>
  <a href="publication.html">
    <span class="fa fa-university"></span>
     
    Publication
  </a>
</li>
<li>
  <a href="cv.html">
    <span class="fa fa-file-pdf-o"></span>
     
    CV
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:magali.richard@univ-grenoble-alpes.fr">
    <span class="fa fa-envelope-o"></span>
     
    Contact me
  </a>
</li>
<li>
  <a href="https://github.com/magrichard">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Research projects</h1>

</div>


<div id="understanding-functional-genetic-diversity-within-regulatory-networks" class="section level2">
<h2>Understanding functional genetic diversity within regulatory networks</h2>
<p>Optimizing growth and survival in face of an environmental signal is a major challenge for single-cell organisms. The yeast galactose regulatory network is a classic model system for studying both cellular and molecular response to an extracellular input. This model offers the possibility to investigate the effects of genetic diversity on a regulatory response at a very fine scale. In order to better understand how natural diversity affects single-cell fates in response to an extracellular input, we are studying natural strains and artificial mutants of Saccharomyces cerevisiae to i) relate genetic diversities to changes in network parameters and ii) identify novel genetic variations involved in the properties of the network.</p>
<p>A Green Fluorescent Protein (GFP) reporter is used to monitor the dynamics of network acquisitions on population of individual cells by flow cytometry. First, this allowed us to unravel a large diversity of responses among natural strains, and to identify natural alleles that can radically change network properties. We are now putting in perspective in vivo experiments with in silico modelling, based on a new model inspired from the literature in order to interpret the functional properties of genetic variants. We intend then to experimentally validate our predictions on the molecular features involved, such as transcription rate, mRNA stability or protein-protein interaction affinities. Second, we are investigating natural wild isolates that display network response bimodality. These two complementary strategies should allow us to characterize natural genetic routes that change a gradual response to a binary response. This project is supported by grant SiGHT StG-281359 from the E.U.</p>
<p><img src="pictures/galONOFF.png" width="800px" style="display: block; margin: auto;" /></p>
</div>
<div id="genomic-profiling-of-cell-proliferation-in-a-periodic-environment" class="section level2">
<h2>Genomic profiling of cell proliferation in a periodic environment</h2>
<p>Experimental approaches to understand the molecular mechanisms of evolution are generally restricted to environmental conditions that are standardized. This limits our comprehension of gene-environment interactions. Several studies indicate that cell-to-cell variability between genetically identical organisms can contribute to fitness in dynamic natural environments. However, such studies are sparse, and our understanding of underlying genetic mechanisms is considerably low.</p>
<p>How fitness varies in natural dynamic environments, how genes deal with external fluctuations, and whether variability is a natural mechanism to anticipate changes remain largely unclear. As a systematic and unbiased approach, we try to decrypt how cells cope with fluctuating environments by i) characterizing the genetic roots of adaptability to dynamic environments and the potential genotype-environment interactions and ii) studying the potential fitness benefit of cell-to-cell heterogeneity in natural contexts.</p>
<p>We have developed an experimental screen to identify genotypes improving proliferation in fluctuating environments. Genetically heterogeneous yeast populations are grown in competition, in various dynamic environments and quantified at several time points. All the yeast strains that we are using have a barcode tag inserted in the genome that allows us to quantify them by PCR and sequencing (BAR-seq). Custom multiplexing allows us to analyse 1,000 competing populations in a single run. To normalize our BAR-seq data, we used the method proposed in the DESeq package for RNA-seq. We are currently implementing a Generalized Linear Model method to identify strains that have a fitness advantage specifically in dynamic environment. This project is supported by grant SiGHT StG-281359 from the E.U.</p>
<p><img src="pictures/pleiotropy.png" width="800px" style="display: block; margin: auto;" /></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
